packages/@aws-cdk/aws-cognito/lib/user-pool-attr.ts

```diff
@@ -341,3 +341,250 @@ export class DateTimeAttribute implements ICustomAttribute {
     };
   }
 }
+
+/**
+ * This interface contains all standard attributes recognized by Cognito
+ * from https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-attributes.html
+ * including `preferred_email` and `preferred_phone_number`
+ */
+export interface StandardAttributesMask {
```

Comment:
There are two sets of standard attributes now (this and the `StandardAttributes` interface).

Either we should consolidate them into one place, or at least, have some kind of test that ensures that they stay consistent.

Reply:
I've added a test case checking whether all attributes in `StandardAttributeNames` are supported in `ClientAttributed` (i.e. are present in `StandardAttributesMask`).
see https://github.com/tom139/aws-cdk/blob/6b6630caa6dfa88a47960e8bcd0d211f46b6d521/packages/%40aws-cdk/aws-cognito/test/user-pool-attr.test.ts#L270-L289

Reply:
Nice!